app/views/menu.ejs

```diff
@@ -15,16 +15,99 @@
         <%- include partials/navbar.ejs %>
 
         <!-- Jumbotron -->
-        <div class="jumbotron" style="background-size: cover;">
-            <div class="container" style="width: 35%; margin-top: 5%; padding: 5%; background-color: white; border-radius: 2%;">
-                <h5 style="text-align: center;">Order Food Online Now!</h5>
-                <input class="form-control" type="search" placeholder="Search for a dish...">
-            </div>
-        </div>       
+        <div class="main-content-container">
+            <div class="jumbotron" style="background-size: cover;">
+                <div class="container" style="width: 35%; margin-top: 5%; padding: 5%; background-color: white; border-radius: 2%;">
+                    <h5 style="text-align: center;">Order Food Online Now!</h5>
+                    <input class="form-control" type="search" placeholder="Search for a dish...">
+                </div>
+            </div>   
+
+            <div class="lower-content-container">
+                <!-- Sidebar -->
+                <div class="lower-left-container">
+                    <div id="sidebar">
+                        <ul class="nav">
+                        <li>
+                            <a href="#">
+                            Whole Menu
+                            </a>
```

Comment:
You don't need to worry about these being links.  Clicking them won't take us to another page or link, it will just filter what we can see on the current page.  These can just be <li> boxes that contain text that we can eventually give functions by checking whether a user clicks on them.  So for each of these you can remove anything relating to the <a> tag.